compaction/size_tiered_compaction_strategy.hh

```diff
@@ -82,7 +82,15 @@ public:
     }
 #endif
     static void validate(const std::map<sstring, sstring>& options, std::map<sstring, sstring>& unchecked_options);
-
+private:
+    static long validate_sstable_size(const std::map<sstring, sstring>& options);
```

Comment:
Also here, I don't think you need all these low-level functions in the header file, it would have been nicer (I think) all of them were just static utility functions in the .cc file.